packages/node-cli/src/core/sdksetup/SdkDownloadService.js

```diff
@@ -41,52 +45,73 @@ class SdkDownloadService {
 		const sdkDirectory = this._fileSystemService.createFolder(sdkParentDirectory, FOLDERS.NODE_CLI);
 
 		const fullURL = `${SdkProperties.getDownloadURL()}/${SdkProperties.getSdkFileName()}`;
+		const destinationFilePath = path.join(sdkDirectory, SdkProperties.getSdkFileName());
```

Comment:
I think this 3 variables are only used inside downloadJarFilePromise. 
Could we define them there? Or do you think it is better to define it here.

In special proxy and skipProxy, because if you don´t pass destinationFilePath you would have to pass sdkDirectory, so it´s fine with it.

Reply:
I want to make the `downloadJarFilePromise` method more "clean", and to take care only of downloading the jar file or throw/reject when it's not possible.
I think that the logic to get the info about the  proxy, destinationFile and downloadUrl shouldn't be part of the download function/service.